### DISCUSSION TITLE
Location and Canendar on new line

### DISCUSSION THREAD

**Discussion by jtronicus** [totaldebug/atomic-calendar-revive#1091] [Support]
Is there a way to move the location and calendar entities to a new line, similar to the hoursOnSameLine config option?

Instead of looking like this:
![image](https://github.com/totaldebug/atomic-calendar-revive/assets/46967605/bcd01cf1-ac37-45e5-aedb-43fbf614b3de)

I am trying to make it look like this:
![image](https://github.com/totaldebug/atomic-calendar-revive/assets/46967605/d5a6cf8e-0c63-4257-b3e0-01b2f47c94ca)

I dont see a config option for this, and I am not sure if this is something card-mod can do.

**Comment by marksie1988** [comment]
The events use the following divs, by default event-main is to the left and event-location is to the right. im sure that it would be possible to manipulate the CSS to change the location of these.  

```
<div class="event-main"></div>
<div class="event-location"></div>
```

im not the best with CSS so not so sure on how this could be achieved.
